src/Backoffice/Airline/App/Controllers/StoreAirlineController.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Lightit\Backoffice\Airline\App\Controllers;
+
+use Illuminate\Http\JsonResponse;
+use Lightit\Backoffice\Airline\App\Requests\UpsertAirlineRequest;
+use Lightit\Backoffice\Airline\App\Resources\AirlineResource;
+use Lightit\Backoffice\Airline\Domain\Actions\UpsertAirlineAction;
+
+class StoreAirlineController
+{
+    public function __invoke(
+        UpsertAirlineRequest $upsertAirlineRequest,
+        UpsertAirlineAction $upsertAirlineAction,
+    ): JsonResponse {
+        $airline = $upsertAirlineAction->execute($upsertAirlineRequest->toDto());
```

Comment:
```suggestion
        UpsertAirlineRequest $request,
        UpsertAirlineAction $action,
    ): JsonResponse {
        $airline = $action->execute($request->toDto());
```
Same thing here as well. Being overly specific is more useful when you have more than a single request (which in reality you wouldn't have more than one) and action. Additionally, you'll notice that Laravel's standard is to name the request variable `$request` (also this is what you'll see in projects).